tests/cmd/atom.md

```diff
@@ -0,0 +1,16 @@
+```console
+$ cobalt -v build --destination _dest
```

Comment:
We need `.out` directories made so we can snapshot the filesystem.

`TRYCMD=dump` will dump to a directory.  You can then copy the output over into the test